src/promptflow-devkit/promptflow/_sdk/operations/_local_storage_operations.py

```diff
@@ -105,19 +105,22 @@ def __enter__(self):
             else:
                 log_path.touch()
 
-        for _logger in self._get_execute_loggers_list():
+        # set the log level for all loggers except the logger "promptflow"
+        _loggers = [_logger for _logger in self._get_execute_loggers_list() if _logger.name != logger.name]
```

Comment:
What's the behavior when stream is False

Reply:
Offline synced. Will print debug logs in terminal if debug is True.

Reply:
If there is no `--debug`, then it's the same with before. If there is `--debug` or `--verbose`, then there will be additional logs.